auth-request.lua

```diff
@@ -67,7 +67,7 @@ core.register_action("auth-request", { "http-req" }, function(txn, be, path)
 	-- are not `DOWN`.
 	local addr = nil
 	for name, server in pairs(core.backends[be].servers) do
-		if server:get_stats()['status'] ~= "DOWN" then
+		if server:get_stats()['status'] == "UP" then
```

Comment:
This breaks for backends without any health checks at all. The status is `no check` for these. Now looking into the source code of haproxy there appears to be `UP x/y` (UP, going down) as well.

This probably should look something like this (trying out the new suggestion feature, but please check whether this works for you):
```suggestion
		local status = server:get_stats()['status']
		if status == "no check" or status:find("UP") == 1 then
```